cli.js

```diff
@@ -8,6 +8,7 @@ var argv = require('optimist')
 .describe('cross-platform', 'do not archive platform-specific files in node_modules')
 .describe('incremental-install', 'start npm install with last node_modules instead of clearing them')
 .describe('production', 'start npm install with production flag')
+.describe('fullText', 'Sha-1 calculated from full text of package.json')
```

Comment:
`fullText` is associated with Full Text Search for many people, this may be a bit confusing.
Do you have another name in mind?
Maybe `full-package-json-sha1`?
Also, for consistency, could you name it here sausage-case?